pkg/commands/compute/deploy.go

```diff
@@ -445,7 +445,8 @@ type activator func(customerID string) error
 func preconfigureActivateTrial(endpoint, token string, httpClient api.HTTPClient) activator {
 	return func(customerID string) error {
 		path := fmt.Sprintf(undocumented.EdgeComputeTrial, customerID)
-		_, err := undocumented.Get(endpoint, path, token, httpClient)
+		var body io.Reader // nil: no body needed to be sent
+		_, err := undocumented.Call(endpoint, path, http.MethodPost, token, body, httpClient)
```

Comment:
Readability. 

I dislike looking at an unfamiliar function and wondering what a `nil` represents 🙂 

I admit it's not the end of the world having the `nil` inlined (because I know I can find it out very quickly) but the variable also lets me document that there is no need to send a body to whatever `Call` is.

I guess I could improve this by having `Call` accept an options struct and then that would...

A.) Allow the code to be clearer as I'm naming the fields.
B.) Allow me to just omit the Body altogether and nil will automatically be inferred.

Reply:
Fixed in [3d0ff37](https://github.com/fastly/cli/pull/927/commits/3d0ff372d48e7d7552596fb9d80209d61fa234de)

Reply:
The readability makes sense. Although when I read it, I assume that the body is non-nil since if it were me I would have used `nil`. Can go either way here, but I think your point about using an options struct is valid.